src/components/pages/events/myEvents/MyEvents.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect, useContext } from 'react';
+import { Event } from 'models/apiModels';
+import { getJoinedEvents } from 'api';
+import styles from './MyEvents.module.scss';
+import { AuthenticateContext } from 'contexts/authProvider';
+import MyEventCard from 'components/molecules/event/myEventCard/myEventCard';
+
+const MyEvents = () => {
+  const [events, setEvents] = useState<Event[]>([]);
+  const { authenticated } = useContext(AuthenticateContext);
+
+  /* Detect error to display message */
+  const [isErr, setIsErr] = useState<boolean>(false);
+
+  /* Fetch joined events on auth update */
+  useEffect(() => {
+    fetchEvents();
+  }, [authenticated]);
+
+  /* Sort events by date (TODO: --> utils/sorting) */
+  const sortEventByDate = (a: Event, b: Event) => {
+    return Number(new Date(a.date)) - Number(new Date(b.date));
+  };
+
+  const fetchEvents = async () => {
+    try {
+      /* Get events */
+      const events = await getJoinedEvents();
+      const sorted = [...events].sort(sortEventByDate);
+      setEvents(sorted);
+    } catch (error) {
+      console.log(error.statusCode);
```

Comment:
Remove🔥

Reply:
Don't we want to console log error text returned from API?

Reply:
IMO no. I think we should report all errors as toasts to the user, and in cases where it's not necessary to report to the user, we can just do nothing